### PR TITLE
Small fixes to the release guide

### PR DESCRIPTION
**Pull request by afrittoli**
I've been running through the release guide, and fixed a few minor
issues.

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [ ] Includes [tests](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if functionality changed/added)
- [ ] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [ ] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/pipeline/blob/master/CONTRIBUTING.md) for more details._

Double check this list of stuff that's easy to miss:

- If you are adding [a new binary/image to the `cmd` dir](../cmd), please update
  [the release Task](../tekton/publish.yaml) and [TaskRun](../tekton/publish-run.yaml) to build and release this image

## Reviewer Notes

If [API changes](https://github.com/tektoncd/pipeline/blob/master/api_compatibility_policy.md)
are included, [additive changes](https://github.com/tektoncd/pipeline/blob/master/api_compatibility_policy.md#additive-changes)
must be approved by at least two [OWNERS](https://github.com/tektoncd/pipeline/blob/master/OWNERS)
and [backwards incompatible changes](https://github.com/tektoncd/pipeline/blob/master/api_compatibility_policy.md#backwards-incompatible-changes)
must be approved by [more than 50% of the OWNERS](https://github.com/tektoncd/pipeline/blob/master/OWNERS),
and they must first be added
[in a backwards compatible way](https://github.com/tektoncd/pipeline/blob/master/api_compatibility_policy.md#backwards-compatible-changes-first).

### CODE REVIEW

File: tekton/README.md [hunk 1]
```diff
@@ -37,9 +36,9 @@ To make a new release:
 1. Create the new tag and release in GitHub
    ([see one of way of doing that here](https://github.com/tektoncd/pipeline/issues/530#issuecomment-477409459)).
    _TODO(#530): Automate as much of this as possible with Tekton._
-1. Add an entry to [the README](../README.md) at `HEAD` for docs and examples for the new release
-   ([README.md#read-the-docs](README.md#read-the-docs)).
-1. Update the new release in GitHub with the same links to the docs and examples, see
+4. Add an entry to [the README](../README.md) at `HEAD` for docs and examples for
```
Comment (gavinfish) [inline tekton/README.md]: Automatically changed numbering? Several similar issues here.

File: tekton/README.md [hunk 2]
```diff
@@ -86,17 +86,22 @@ To use [`tkn`](https://github.com/tektoncd/cli) to run the `publish-tekton-pipel
...
    # Be careful: due to #983 it is possible to overwrite previous releases.
    export VERSION_TAG=v0.X.Y
+   export IMAGE_REGISTRY=gcr.io/tekton-releases
+
+   # Double-check the git revision that is going to be used for the release:
+   kubectl get pipelineresource/tekton-pipelines-git -o=jsonpath="{'Target Revision: '}{.spec.params[?(@.name == 'revision')].value}{'\n'}"
 
    tkn pipeline start \
 		--param=versionTag=${VERSION_TAG} \
+    --param=imageRegistry=${IMAGE_REGISTRY} \
```
Comment (gavinfish) [inline tekton/README.md]: Align with other parameters?

File: tekton/README.md (hunk 1 above)
Comment (afrittoli) [reply tekton/README.md]: thanks!

File: tekton/README.md (hunk 2 above)
Comment (afrittoli) [reply tekton/README.md]: Nice catch, thx

**Pull request merged**